sevntu-checks/src/main/resources/com/github/sevntu/checkstyle/checks/coding/messages.properties

```diff
@@ -19,6 +19,8 @@ finalize.implementation.public = finalize() method should have a "protected" vis
 finalize.implementation.useless = finalize() method is useless: it does nothing except for calling super.finalize().
 finalize.implementation.missed.super.finalize = You have to call super.finalize() right after finally opening brace.
 forbid.certain.imports=Import ''{1}'' should not match ''{0}'' pattern, it is forbidden.
+forbid.certain.method=Call to ''{0}'' method (matches pattern {1}) with ''{2}'' arguments (matches pattern {3}) is forbidden.
+forbid.certain.method.noarg=Call to ''{0}'' method (matches pattern {1}) is forbidden.
```

Comment:
I think "forbid.certain.method.noarg" should be dropped as rage is always exists, even it is default.